articles/iot-hub/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringV2.md

```diff
@@ -0,0 +1,173 @@
+
+## What you need
+
+Finish the [Getting Started Guide](https://docs.microsoft.com/azure/iot-hub/iot-hub-arduino-iot-devkit-az3166-get-started) to:
+
+* Have your DevKit connected to Wi-Fi
+* Prepare the development environment
+
+An active Azure subscription. If you do not have one, you can register via one of these two methods:
+
+* Activate a [free 30-day trial Microsoft Azure account](https://azureinfo.microsoft.com/us-freetrial.html)
+* Claim your [Azure credit](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/) if you are MSDN or Visual Studio subscriber
+
+## Create an Azure IoT Suite
+
+1. Go to [Azure IoT Suite site](https://www.azureiotsuite.com/) and click **Create a new solution**.
+
+  ![Select Azure IoT Suite type](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-v2-solution-types.png)
+ 
+
+2. Select **Remote monitoring preview**.
+
+3. Enter a solution name, select a subscription and a region, and then click **Create solution**. The solution may take a while to be provisioned.
+
+  ![Create solution](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-create-remote-monitoring-preview-solution.png)
+
+
+4. After the provision is completed, click **Launch**. Some simulated devices are created for the solution during the provision process. Click **DEVICES** to check them out.
+
+  ![Dashboard](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-new-solution-created.png)
+
+  ![Console](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-console.png)
+
+
+
+## Open the RemoteMonitoring sample
+
+1. Disconnect the DevKit from your computer, if it is connected.
+
+2. Start VS Code.
+
+3. Connect the DevKit to your computer. VS Code automatically detects your DevKit and opens the following pages:
+  * The DevKit introduction page.
+  * Arduino Examples: Hands-on samples to get started with DevKit.
+
+4. Expand left side **ARDUINO EXAMPLES** section, browse to **Examples for MXCHIP AZ3166 > AzureIoT**, and select **RemoteMonitoringv2**. It opens a new VS Code window with a project folder in it.
+
+  ![Open remote monitoring project](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-arduino-examples.png)
+
+
+  > [!NOTE]
+  > If you happen to close the pane, you can reopen it. Use `Ctrl+Shift+P` (macOS: `Cmd+Shift+P`) to open the command palette, type **Arduino**, and then find and select **Arduino: Examples**.
+
+## Provision required Azure services
+
+In the solution window, run your task through `Ctrl+P` (macOS: `Cmd+P`) by entering `task cloud-provision` in the provided text box:

```

Comment:
For RemoteMornitoring, the service has been provisioned in the portal, so no need to run this task (we will also remove this task later)

Reply:
this section can be removed entirely